### DISCUSSION TITLE
Please help: AttributeError: module 'torch' has no attribute 'dml'

### DISCUSSION THREAD

**Discussion by Octahead** [lshqqytiger/stable-diffusion-webui-directml#400] [Q&A] [answered]
Hi, any help would be much appreciated!

I am getting this error message:
>>AttributeError: module 'torch' has no attribute 'dml'<<

webui-user.bat
@echo off

set PYTHON=
set GIT=
set VENV_DIR=
set COMMANDLINE_ARGS= --skip-torch-cuda-test --use-directml

call webui.bat

What am I doing wrong?

![Error](https://github.com/lshqqytiger/stable-diffusion-webui-directml/assets/34001627/07f40317-4ddc-4ffd-b8ce-cb7c8546f8d1)

**Comment by Octahead** [comment, marked as answer]
I solved the problem by deleting the venv folder and only using --use-directml.

**Reply by HycasOrange** [reply]
Thank you so much for finding that, just generated an image in like 8 seconds that woulda taken at least a minute before.
